src/clincoded/static/components/provisional_classification/snapshots.js

```diff
@@ -266,7 +287,7 @@ class CurationSnapshots extends Component {
                                 </td>
                                 <td className="approval-snapshot-buttons">
                                     {this.renderSnapshotStatusIcon(snapshot, 'Provisioned')}
-                                    {resourceParent && !isApprovalActive && classificationStatus !== 'Approved' ?
+                                    {resourceParent && !this.props.isPublishEventActive && !isApprovalActive && classificationStatus !== 'Approved' ?
```

Comment:
Why not put all these conditions into the `renderProvisionalSnapshotApprovalLink()` function?

Reply:
I didn't move all of these conditions because I was simply adding one to an existing list.

Reply:
The latest changes look great. Thanks!